test/__snapshots__/test.js.snap

```diff
@@ -1,5 +1,75 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`with processors AsyncCallbackInterface.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const interfaceName = \\"AsyncCallbackInterface\\";
+
+exports.convert = function convert(value, { context = \\"The provided value\\" } = {}) {
+  if (!utils.isObject(value)) {
+    throw new TypeError(\`\${context} is not an object.\`);
+  }
+
+  const { wrapper } = {
+    wrapper(...args) {
+      let completion;
```

Comment:
This variable appears to be unused.

Reply:
That’s a leftover from when I was comparing different strategies on how to handle callback interfaces that return `Promise`s.